benchmarks/decomposition.py

```diff
@@ -34,6 +39,60 @@ def peakmem_fit(self, *args):
 
 class DictionaryLearning_bench(Benchmark):
     """
-    Benchmarks for DictionnaryLearning.
+    Benchmarks for DictionaryLearning.
+    """
+
+    # params = (fit_algorithm)
+    param_names = ['params'] + Benchmark.param_names
+    params = ([('lars', ), ('cd', )], ) + Benchmark.params
+
+    def setup(self, params, *common):
+        self.data = _decomposition_dataset()
+        fit_algorithm = params[0]
+        n_jobs = common[0]
```

Comment:
and here n_jobs is passed differently

Reply:
I just followed the same pattern here as in the benchmarks by Jeremy. Right now everything is consistent.
Basically, if we have any additional params, we reassign `params` and pass extra `common` list with default params set in Benchmark class.